tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -118,12 +119,30 @@ stages:
                   - checkout: none
                   - download: current
                     artifact: release-artifacts
+                  - pwsh: |
+                      $latestVersion = (azcopy list "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp/") -replace '[\/;].*' | Get-Unique | sort {[version] $_} -Descending | Select-Object -First 1
```

Comment:
Isn't the version number somewhere in the source code?

Reply:
no, the version number was determined by the universal package devops step